lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -1,5 +1,29 @@
 <%! from django.utils.translation import ugettext as _ %>
 <%namespace name='static' file='../static_content.html'/>
+<%block name="js_extra">
+  <%static:js group='certificates_wv'/>
+  <script type="text/javascript">
+    $(document).ready(function() {
+      $.ajaxSetup({
+        headers: {
+          'X-CSRFToken': $.cookie('csrftoken')
+        },
+        dataType: 'json'
+      });
+      $(".action-linkedin-profile").click(function() {
+        var data = {
+          user_id: '${accomplishment_user_id}',
+          course_id: $(this).data('course-id'),
+          enrollment_mode: $(this).data('certificate-mode'),
+          certificate_id: '${certificate_id_number}',
+          certificate_url: window.location.href,
+          social_network: 'LinkedIn'
+        };
+        Logger.log('edx.certificate.shared', data);
```

Comment:
@mattdrayer @mulby added certificate web view page and bok choy test to test `edx.certificate.shared` event is emitted.